var/spack/repos/builtin/packages/stat/package.py

```diff
@@ -76,9 +81,10 @@ class Stat(AutotoolsPackage):
     variant("gui", default=True, description="enable GUI")
 
     depends_on("autoconf", type="build")
-    depends_on("automake@:1.15", type="build")
+    depends_on("automake", type="build")
     depends_on("libtool", type="build")
-    depends_on("dyninst@:11.9", when="~dysect")
+    depends_on("dyninst", when="@4.2.1:")
```

Comment:
Just a note that this could just be written as:
```suggestion
    depends_on("dyninst")
```
to cover for the most generic statement "stat depends on dyninst", without further constraints.